gemrb/GUIScripts/GUICommon.py

```diff
@@ -558,6 +558,20 @@ def IsMultiClassed (actor, verbose):
 	# return the tuple
 	return (NumClasses, Classes[0], Classes[1], Classes[2])
 
+def IsNamelessOne (actor):
+	# simple check to identify if the actor is TNO
+	# returns his active class if true
+
+	if not GameCheck.IsPST():
+		return False
+
+	Specific = GemRB.GetPlayerStat (actor, IE_SPECIFIC)
+	if Specific == 2:
+		Class = GetClassRowName(actor)
+		return Class
+
+	return False
+
```

Comment:
the name `IsNamelessOne` suggest a boolean return value which is not the case. I would return `None` instead of `False`.

It's also not clear to me how you can hit the last return. shouldn't it be an exception if `Specific != 2`? maybe I just dont understand PST leveling tho.

Reply:
Re 2: no, other pcs will run this code as well.